driver/api/impl/impl.cpp

```diff
@@ -0,0 +1,1284 @@
+#include "driver/api/impl/impl.h"
```

Comment:
Separating the implementation and the API calls bodies, and then consolidating implementations together (in `impl.cpp`), and API bodies together (in `odbc.cpp`). My experience with this driver's code shows that it is very misleading to have things separated into these files (attr.cpp, handles.cpp, etc.) - you usually don't know which function will have the piece of code you are looking for, and where that function is "classified" according to this classification. It turned out to be much more maintainable to have them, at least at this point, classified by their logical nesting level, instead of what they do according their names.